app/models/user.rb

```diff
@@ -9,6 +9,10 @@ class User < ApplicationRecord
   validates :email, presence: true, uniqueness: true,
     length: {maximum: Settings.validator.user.email.max_length},
     format: {with: VALID_EMAIL_REGEX}
+  validates :password, presence: true,
+  length: {minimum: Settings.validator.user.pass.pass_min}, allow_nil: true
```

Comment:
```suggestion
    length: {minimum: Settings.validator.user.pass.pass_min}, allow_nil: true
```

allow_nil: true -> giải thích code này nghe coi

Reply:
Là khi mà mình cập nhật thông tin người dùng thì không cần phải nhập mật khẩu

Reply:
trả lời ri ah cậu
đang bảo giải thích code `allow_nil: true` mà

Reply:
allow_nil: true Nó sẽ cho phép bỏ qua validation khi giá trị là nil